index.src.html

```diff
@@ -4870,7 +5073,7 @@
 6. Return `true`.
 
 A <dfn>valid image MIME type</dfn> is a string that is a [=valid MIME type
-string=] and for which the `type`, per Section 3.1.1.1 of [[RFC7231]], is
+string=] and for which the `type`, per Section 3.1.1.1 of [[RFC9110]], is
```

Comment:
Was this a typo before?

Reply:
Bikeshed let me know that the original RFC was obsoleted by this new one.